apps/heureka/src/lib/slices/createGlobalsSlice.js

```diff
@@ -124,13 +124,19 @@ const createGlobalsSlice = (set, get, options) => ({
         ),
 
       // Sync details state with the URL
-      syncDetailsWithURL: () => {
-        const state = get().globals
+      syncDetailsWithURL: (serviceDetail, issueDetail, panelType) => {
```

Comment:
Here’s a proposal for the future: since this function uses the globals internally without modifying anything in the store, it might be better to refactor it into a helper function where globals are passed as a parameter. What do you think?